### PR TITLE
Report manager

### PR DESCRIPTION
**Pull request by Tacchinardi20**

### CODE REVIEW

File: 0002_child_test.js [hunk 1]
```diff
@@ -321,23 +347,23 @@ describe("Test Social Worker Note:", () => {
       cy.log(
```
Comment (Achoobert) [inline 0002_child_test.js:347]: According to this comment:
If the popup is closing, these should be deleted as part of the change that fixes the proble

File: 0002_child_test.js [hunk 2]
```diff
@@ -378,32 +383,31 @@ describe("Test Social Worker Note:", () => {
...
-         .filter(":visible")
-         .contains("Close")
-         .click({ force: true, multiple: true });
-      cy.get(
-         cyInterfaceCARS.page.socialWorker.page.children.view.child.page
-            .socialWork.page.notes.form.addNote.field.categories
-      )
-         .find(".webix_multicombo_input")
-         .click();
+      // cy.get(".webix_button")
```
Comment (Achoobert) [inline 0002_child_test.js:386]: Same with these, delete the un-needed code :D

**Review by Achoobert — CHANGES_REQUESTED** [review]

File: 0002_child_test.js (hunk 2 above)
Comment (Tacchinardi20) [reply 0002_child_test.js:386]: OK Isaac, Thank You

**Review by wongpratan — APPROVED** [review]

File: 0001_cars_test.js [hunk 3]
```diff
@@ -624,40 +632,78 @@ describe("Test Child:", () => {
          .type("a")
          .clear()
          .type("test");
-      cy.get('[data-cy^="connectObject Child"]').should("not.exist");
+      cy.get('[data-cy^="connectObject Child"]').should("exist").click();
```
Comment (Achoobert) [inline 0001_cars_test.js:635]: Does a `should("be.visible")` work here?

**Review by Achoobert — APPROVED** [review]
LGTM

**Pull request merged**